headers.js

```diff
@@ -52,7 +52,7 @@ const cspMeta = Object.entries({
     process.env.REACT_APP_UNCHAINED_BITCOIN_HTTP_URL,
     process.env.REACT_APP_UNCHAINED_BITCOIN_WS_URL,
     process.env.REACT_APP_UNCHAINED_COSMOS_HTTP_URL,
-    process.env.REACT_APP_UNCHAINED_COSMOS_WS_URL
+    process.env.REACT_APP_UNCHAINED_COSMOS_WS_URL,
```

Comment:
@0xApotheosis @mrnerdhair should we disable the trailing comma linting rule for this file? i'm not sure if this will break our headers?

Reply:
more context - this file isn't application code per se - it's used at build time, and there's some very specific parsing logic that may get broken by trailing commas

Reply:
I doubt this will break something honestly looking at the process

Reply:
Nope, trailing commas are good here, happy to have them